src/en/tercoscans/src/eu/kanade/tachiyomi/extension/en/tercoscans/TercoScans.kt

```diff
@@ -0,0 +1,35 @@
+package eu.kanade.tachiyomi.extension.en.tercoscans
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class TercoScans : MangaThemesia(
+    "Terco Scans",
+    "https://tecnoxcomics.xyz",
```

Comment:
redirects to `https://tercofable.xyz/`

Reply:
By the way, for myself, tercofable redirect to tecnoxcomics